components/streamer-list.vue

```diff
@@ -0,0 +1,41 @@
+<script lang="ts" setup>
+import { useStreamersStore } from '@/stores/streamersStore';
+
+const streamersStore = useStreamersStore();
+
+const streamers = ref();
+
+onMounted(async () => {
+  const res = await streamersStore.getStreamers();
+  streamers.value = streamersStore.streamers;
+});
+</script>
+<template>
```

Comment:
Add a space between script and template
```suggestion

<template>
```